tests/unit_tests/test_game.py

```diff
@@ -315,3 +377,19 @@ def test_hashing(game):
     await game.launch()
 
     assert game.players == frozenset(players)
+
+def test_victory_conditions():
+    conds = [("demoralization", Victory.DEMORALIZATION),
+             ("domination", Victory.DOMINATION),
+             ("eradication", Victory.ERADICATION),
+             ("sandbox", Victory.SANDBOX)]
+
+    assert (Victory.from_gpgnet_string(t[1]) == t[2] for t in conds)
```

Comment:
This assertion will always be true, since you're asserting truthyness of a generator object.

Instead unfold the loop (And again; use a list of tuples):
```python
for string_value, enum_value in conds:
    assert Victory.from_gpgnet_string(string_value) == enum_value
```